__tests__/components/Card-test.js

```diff
@@ -0,0 +1,62 @@
+// (C) Copyright 2014-2016 Hewlett Packard Enterprise Development LP
+
+import React from 'react';
+import renderer from 'react/lib/ReactTestRenderer';
+
+import Card from '../../src/js/components/Card';
+import FakeIcon from '../mocks/FakeIcon';
+import Anchor from '../../src/js/components/Anchor';
```

Comment:
Had to remove the following test:
```
  it('renders a video component correctly', () => {
    const component = renderer.create(
      <Card direction="row"
        thumbnail="/img/carousel-1.png" label="Video - 4:27"
        heading="Foundation Paraguay Empowers Microbusinesses"
        text="See how Hewlett Packard Enterprise delivers mobile
            solutions to improve quality of life and help eliminate poverty
            in South America."
        video={{"source": "/video/test.mp4", "type": "mp4"}}
        link={<Anchor href="#" label="Watch Now" icon={<FakeIcon />} />} />
    );
```
due to the fact that one of the SVG elements was not matching the snapshot.  If someone more familiar with the project wants to investigate it, please do!